scripts/kani.py

```diff
@@ -431,8 +431,8 @@ def run_goto_instrument(
 
 # Processes vtable restrictions to the format CBMC expects
 def process_vtable_restrictions(restrictions_filename, verbose=False, dry_run=False):
-    cmd = ["./target/release/kani-link-restrictions"]
-    outname = os.path.join(os.path.dirname(os.path.abspath(restrictions_filename)), "linked-restrictions.json")
+    cmd = ["/home/ubuntu/rmc/target/release/kani-link-restrictions"]
```

Comment:
Probably forgot to update this.

Reply:
Yeah, I'm trying to just get things through regression on my local machine. This PR won't be ready for review/merge until a PR is merged into cbmc and released and fully available. It'll be a few weeks!